### ISSUE TITLE
Caused by: java.lang.NullPointerException in DocumentAssembler

### ISSUE THREAD

**Issue by maziyarpanahi** [JohnSnowLabs/spark-nlp#941]
If the input is empty it’s possible DocumentAssembler crashes with NullPointer exception.

```

Caused by: java.lang.NullPointerException
	at com.johnsnowlabs.nlp.DocumentAssembler.assemble(DocumentAssembler.scala:98)
	at com.johnsnowlabs.nlp.DocumentAssembler$$anonfun$dfAssembleNoExtras$1.apply(DocumentAssembler.scala:124)
	at com.johnsnowlabs.nlp.DocumentAssembler$$anonfun$dfAssembleNoExtras$1.apply(DocumentAssembler.scala:123)
	... 67 more
```

**Comment by Ahmetemintek** [comment]
I got the same issue and I haven't solve yet.

**Comment by maziyarpanahi** [comment]
Hi @Ahmetemintek 
Could you please provide more information about the version of the Spark NLP and Apache Spark you are using? Also, have you tried to filter your input to be sure it's not `null` and see if the issue still remains? 
Unfortunately, I couldn't reproduce this error as it depends on the input so if you have a specific example resulting in this error it would be very helpful.

**Comment by Ahmetemintek** [comment]
I am using sparknlp version 2.6.6 and payspark version 2.4.4 
So I created pipeline smoothly like this:
`pipeline= Pipeline(stages=[
    document, 
    tokenizer,
    normalizer,
    stop_words,
    lemma,
    word_embeddings,
    sentence_embeddings,
    classifier  #using by ClassifierDLApproach
])`

 but when I try fitting with my data like this:
`model= pipeline.fit(df_train)`
 I got this issue which you show above. 
Then,   my data has some null values as you said me, however I cleaned the null values from data like this:
`df_train= df_train.na.drop(how="any")`

But now, I got this issue:
"requirement failed: We couldn't find any suitable graph for 2000 classes."
Why have I got this? I don't have any idea.

**Comment by maziyarpanahi** [comment]
- Please make sure you have `2.6.2` latest release. (we don't have 2.6.6)
- The first issue is resolved if you don't have any null document.
- Your second issue is that you are trying to train something that doesn’t support that many classes. 

You are saying `ClassifierDLApproach` annotator, however, the message is coming from `ContextSpellCheckerApproach`. In either case, the ClassifierDLApproach only accepts up to 100 classes and here for more examples: 
https://github.com/JohnSnowLabs/spark-nlp-workshop/tree/master/jupyter/training/english/classification

If you are trying to train a spell checker `ContextSpellCheckerApproach`, you can follow this example: https://github.com/JohnSnowLabs/spark-nlp-workshop/blob/master/jupyter/training/italian/Training%20Context%20Spell%20Checker%20-%20Italian.ipynb

Since the first issue is resolved, please open a new issue following the template for us to reproduce it (if any) and we can continue there as I will close this issue once it's resolved in the new release. (workaround is to make sure there is no null value, but I'll try to skip it with an empty Document in the next release)

**Comment by maziyarpanahi** [comment]
I observed this issue in the `2.6.3` release, I will investigate more.

**Comment by maziyarpanahi** [comment]
Fixed in `2.6.4` release: https://github.com/JohnSnowLabs/spark-nlp/releases/tag/2.6.4
